tests/test_int64_tensor.py

```diff
@@ -3,23 +3,21 @@
 import numpy as np
 import tensorflow as tf
 import tensorflow_encrypted as tfe
-from tensorflow_encrypted.tensor.int64 import Int64Factory, Int64Tensor
+
+from tensorflow_encrypted.tensor import int64factory, fixed64
 
 
 class TestInt64Tensor(unittest.TestCase):
     def setUp(self):
         tf.reset_default_graph()
 
     def test_pond(self) -> None:
-        config = tfe.LocalConfig([
-            'server0',
-            'server1',
-            'crypto_producer'
-        ])
-
-        with tfe.protocol.Pond(*config.get_players('server0, server1, crypto_producer'),
-                               tensor_factory=Int64Factory(), use_noninteractive_truncation=True,
-                               verify_precision=False) as prot:
+
```

Comment:
could use more comprehensive testing, perhaps we can make a separate issue about it